app/src/main/java/com/example/trashminder/utils/AlarmReceiver.kt

```diff
@@ -0,0 +1,84 @@
+package com.example.trashminder.utils
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import android.util.Log
+import androidx.annotation.RequiresApi
+import androidx.core.app.NotificationCompat
+import com.example.trashminder.R
+import java.util.Calendar
+import java.util.concurrent.TimeUnit
+
+class AlarmReceiver : BroadcastReceiver() {
+    @RequiresApi(Build.VERSION_CODES.M)
+    override fun onReceive(context: Context, intent: Intent?) {
+
+        val manager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+        val chanelName = "message_channel"
+        val channelId = "message_id"
```

Comment:
A private const val could be used for this. E.g: `private const val CHANNEL_ID = "TrashRemindersChannel"`